include/SFML/System/Angle.inl

```diff
@@ -28,7 +28,7 @@ constexpr float pi = 3.141592654f;
 
 constexpr float positiveRemainder(float a, float b)
 {
-    assert(b > 0.0f);
+    assert(b > 0.0f && "Cannot calculate non-positive remainder");
```

Comment:
This description doesn't look right. Maybe something like this would be more correct: "Computing remainder with non-positive divisor is not allowed".
By the way I am not sure why we have this restriction.